### PR TITLE
Port over AppVeyor support to master

### LINKED ISSUE #210: No CI for master branch
All of the AppVeyor work was implemented only on the fsharp4 branch

It needs to be ported to master.

### PR DESCRIPTION
**Pull request by latkin**
Fixes #210.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![Issue Stats](http://issuestats.com/github/Microsoft/visualfsharp/badge/pr)](http://issuestats.com/github/microsoft/visualfsharp)
 [![Issue Stats](http://issuestats.com/github/Microsoft/visualfsharp/badge/issue)](http://issuestats.com/github/microsoft/visualfsharp)
+[![Build status](https://ci.appveyor.com/api/projects/status/sf3s485t5utl31b7/branch/master?svg=true)](https://ci.appveyor.com/project/KevinRansom/visualfsharp-radou/branch/master)
```
Comment (latkin) [inline README.md]: This doesn't seem to be working (at least not yet?). Is there some new api key required?

**Comment by KevinRansom** [comment]
Looks good to me

**Comment by forki** [comment]
Could you please describe the difference between master and fsharp4 in the readme. I'm never 100%  sure to which branch I want to contribute. Thx

**Comment by dsyme** [comment]
LGTM

**Pull request closed without merging**